kratos/tests/test_scipy_conversion_tools.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+try:
+    import KratosMultiphysics.scipy_conversion_tools
+    import numpy as np
+    missing_scipy = False
+except ImportError as e:
+    missing_scipy = True
+
+class TestScipyConversionTools(KratosUnittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_scipy_conversion_tools(self):
+        if not missing_scipy:
```

Comment:
using skiptest would be better in this case IMO, otherwise you will not get informed that sth is not being executed